### PR TITLE
db creation fix in Kapa Tools section

### PR DESCRIPTION
**Pull request by timhallinflux**
adding `skip_database_creation = true` per https://github.com/influxdata/kapacitor/issues/2719

Closes #

_Describe your proposed changes here._

- [X] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [X] Rebased/mergeable

### CODE REVIEW

**Comment by timhallinflux** [comment]
This might require an additional note about pre-creating the bucket/database which is a bit of a change from 1.x.

**Comment by timhallinflux** [comment]
@sanderson @kelseiv -- if you would kindly check my CLA status, I'm positive I signed it.  But... if I need to do it again, I will happily comply.

**Comment by mhall119** [comment]
@timhallinflux I don't see an entry for you in the CLA spreadsheet, so if you wouldn't mind filling it out again: https://www.influxdata.com/legal/cla/

File: content/influxdb/v2.3/tools/kapacitor.md [hunk 1]
```diff
@@ -93,13 +93,16 @@ see [Write back to InfluxDB](#write-back-to-influxdb) below.
 
 ## Use Kapacitor stream tasks
 InfluxDB Cloud and OSS {{< current-version >}} do not have subscription APIs and do not support Kapacitor stream tasks directly.
-To use Kapacitor stream tasks, write data directly to Kapacitor using the [Kapcitior `write` API](/{{< latest "kapacitor" >}}/working/api/#writing-data).
+To use Kapacitor stream tasks, write data directly to Kapacitor using the [Kapacitor `write` API](/{{< latest "kapacitor" >}}/working/api/#writing-data). We recommend using the [Telegraf InfluxDB output plugin](/{{< latest "telegraf" >}}/plugins/#output-influxdb)to write data to both InfluxDB Cloud or OSS and Kapacitor.
```
Comment (kelseiv) [inline content/influxdb/v2.3/tools/kapacitor.md:96]:
```suggestion
To use Kapacitor stream tasks, write data directly to Kapacitor using the [Kapacitor `write` API](/{{< latest "kapacitor" >}}/working/api/#writing-data). We recommend using the [Telegraf InfluxDB output plugin](/{{< latest "telegraf" >}}/plugins/#output-influxdb) to write data to both InfluxDB Cloud or OSS and Kapacitor.
```

**Review by kelseiv — APPROVED** [review]
thank you, @timhallinflux!

**Pull request merged**